test/helpers/GameFlowWrapper.js

```diff
@@ -32,13 +32,11 @@ class GameFlowWrapper {
     }
 
     get firstPlayer() {
```

Comment:
I wonder if we can just remove this, there really isn't a similar concept in swu that's relevant and afaik the method isn't used. There's "acting player" and "player who currently controls initiative," but since initiative can change hands during the round it doesn't carry quite same meaning as "first player."

Iirc the term is actually used once in the rules but it's to determine who gets to choose initiative at the very beginning of the game

Reply:
Makes sense, it's basically a shortcut for initiative player anyway.